src/mscorlib/src/System/Globalization/SortKey.cs

```diff
@@ -55,25 +55,6 @@ internal SortKey(String localeName, String str, CompareOptions options, byte[] k
         }
 
 #if FEATURE_USE_LCID
-        [OnSerializing]
-        private void OnSerializing(StreamingContext context)
-        {
-            //set LCID to proper value for Whidbey serialization (no other use)
-            if (win32LCID == 0)
-            {
-                win32LCID = CultureInfo.GetCultureInfo(localeName).LCID;
-            }
-        }
-
-        [OnDeserialized]
-        private void OnDeserialized(StreamingContext context)
-        {
-            //set locale name to proper value after Whidbey deserialization
-            if (String.IsNullOrEmpty(localeName) && win32LCID != 0)
-            {
-                localeName = CultureInfo.GetCultureInfo(win32LCID).Name;
-            }
-        }
```

Comment:
As per earlier comment, removing these serialization-attributed members will break serialization/deserialization of the type.  We probably have a test gap.